src/ResourceManager/Sql/Commands.Sql/ThreatDetection/Cmdlet/SetAzureSqlServerThreatDetection.cs

```diff
@@ -52,7 +52,6 @@ public class SetAzureSqlServerThreatDetection : SqlServerThreatDetectionCmdletBa
         /// Gets or sets the name of the storage account to use.
         /// </summary>
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of the storage account")]
-        [ValidateNotNullOrEmpty]
```

Comment:
Why was this removed? Would someone provide a storage account name (which is an optional parameter) that is null or empty?

Reply:
The backend now support setting an empty storage account name, but I think I'll revert it and let the threat detection folks define it more properly